apis/keda/v1alpha1/scaledjob_types.go

```diff
@@ -72,6 +72,8 @@ type ScaledJobStatus struct {
 	LastActiveTime *metav1.Time `json:"lastActiveTime,omitempty"`
 	// +optional
 	Conditions Conditions `json:"conditions,omitempty"`
+	// +optional
+	IsPaused string `json:"isPaused,omitempty"`
```

Comment:
Please name this `Paused` see the discussion here: https://github.com/kedacore/keda/issues/3303#issuecomment-1184286650